sycl/test/basic_tests/image_accessor_types.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: not %clangxx -fsyntax-only %s 2>&1 | FileCheck %s
```

Comment:
It is a bit tricky, because error is attached not to this file, but to `accessor.hpp`, where `static_assert` is located.

Experiments showed that `-verify` requires absolute path to file with diagnostic (if it is different from the current file), which seems to be hard to achieve in LIT tests